### ISSUE TITLE
K8s operator and monitoring

### ISSUE THREAD

**Issue by yovanoc** [dragonflydb/dragonfly#2007]
I don't manage to use the operator in k8s with k8s, the doc covers just with the helm chart, thanks

**Comment by romange** [comment]
another good topic to add to our documentation site.

**Comment by Pothulapati** [comment]
Yep, We already have it in https://github.com/dragonflydb/dragonfly-operator/blob/main/monitoring/podMonitorGuide.md, and @Abhra303 is working on moving them to the documentaiton site!

**Comment by yovanoc** [comment]
Oh I see, but please can you add more info on what to change to put the final kind: Dragonfly on a different namespace?
I follow the .md and change the namespace at the end but I think it need more than that
